### ISSUE TITLE
$animate.on: javascript error with IE11 (Angular v1.4.8)

### ISSUE THREAD

**Issue by PhilFlash** [angular/angular.js#13548]
Hi,
$animate.on generates a javascript error with IE11.
All is ok with Chrome and Firefox.
Plunker here (to test with IE):
http://plnkr.co/edit/jtokwmcZRiG0vxkiL6mE?p=preview

Open Developer Tools with F12 and click on "Visible".
The error is:
TypeError: L’objet ne gère pas la propriété ou la méthode "contains"
at Anonymous function (https://code.angularjs.org/1.4.8/angular-animate.js:2146:11)

In line 2146 in angular-animate.js (v1.4.8), there is:
```
function findCallbacks(parent, element, event) {
    ...
	forEach(entries, function(entry) {
          if (entry.node.contains(targetNode)) {
            matches.push(entry.callback);
```
IMHO, this is a problem with method "contains" with IE.
Node.contains is not implemented on Node.
See:
https://connect.microsoft.com/IE/Feedback/Details/785343

But, I haven't found a patch or a workaround for this bug.

Philippe

**Comment by gkalpak** [comment]
This seems to need a similar fix to #10276.

**Comment by petebacondarwin** [comment]
Sounds like it should be a simple fix to use the `contains` like in #10276.
I think this should be fixed before rc.1
